utils.py

```diff
@@ -272,7 +272,14 @@ def convert_png_binary_to_data_url(content):
 
 
 def convert_png_to_data_url(filepath):
-    """Converts the png file at filepath to a data URL."""
+    """Converts the png file at filepath to a data URL.
+
+    Args:
+        filepath: str. A full path to the file.
+
+    Returns:
+        *. Data url created from the filepath of the PNG.
```

Comment:
Why is this `*`? Doesn't this return a string? If so, mention it as `str` here.

Reply:
This function returns a call to the function `convert_png_binary_to_data_url` which has its return type specified like this. Can you make sure whether it is a string so that I can modify docstrings of both the functions? @aks681

Reply:
@aks681 ?